src/ColorScale.js

```diff
@@ -125,7 +126,28 @@ export default class ColorScale extends BaseClass {
 
       const buckets = min([colors ? colors.length : this._buckets, data.length]);
 
-      const jenks = ckmeans(data, buckets);
+      var jenks = [];
+
+      if (diverging && this._centered) {
+        var half = Math.floor(buckets / 2);
+        var residual = buckets % 2;
+
+        var negatives = data.filter(d => d < this._midpoint);
+        var negativesDeviation = deviation(negatives);
+
+        var positives = data.concat(this._midpoint).filter(d => d >= this._midpoint);
+        var positivesDeviation = deviation(positives);
+
+        var isNegativeMax = negativesDeviation > positivesDeviation ? 1 : 0;
+        var isPositiveMax = positivesDeviation > negativesDeviation ? 1 : 0;
+        
+        var negativeJenks = ckmeans(negatives, half + residual * isNegativeMax);
+        var positiveJenks = ckmeans(positives, half + residual * isPositiveMax);
+        
+        jenks = negativeJenks.concat(positiveJenks);
+      } else {
+        jenks = ckmeans(data, buckets);
+      }
```

Comment:
@ffigueroal please take a few minutes to set up `eslint` on your computer/IDE. No `var`s allowed in d3plus, should be using `let` and `const`:

<img width="580" alt="Screen Shot 2020-06-30 at 10 15 04 AM" src="https://user-images.githubusercontent.com/1847581/86136955-ad283c00-baba-11ea-87af-cdd9d145d796.png">

Reply:
@davelandry thanks for the recommendation! I lost `eslint` when changed my laptop and did not installed again.
![image](https://user-images.githubusercontent.com/39899484/86147088-f54d5b80-bac6-11ea-819f-992e65533014.png)